integrations/discord/save.go

```diff
@@ -45,5 +45,5 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	c.Finalize(sdktypes.NewVars().Set(BotToken, r.Form.Get("botToken"), true))
+	c.Finalize(sdktypes.NewVars().Set(botToken, r.Form.Get("BotToken"), true))
```

Comment:
The input field name in Discord's `index.html` file is `botToken`:

```suggestion
	c.Finalize(sdktypes.NewVars().Set(botToken, r.Form.Get("botToken"), true))
```